src/components/Carousel/Carousel.module.css

```diff
@@ -0,0 +1,36 @@
+.carousel {
+  position: relative;
+}
+
+.slides {
+  position: relative;
+}
+
+.hasMultiple {
+  cursor: e-resize;
```

Comment:
General question, are we keeping a similar naming scheme as DR or doing camelCase in this repo?

Reply:
I've tended to prefer camelCase throughout, personally. I feel it makes it much easier to be consistent when naming variables, importing styles, etc. Kebab casing also makes classes semi un-greppable. Happy to hear alternatives, though.

Reply:
I'm fine with camelCase personally, it's easier to keep consistent especially without the `styleName` thing going on